spec/tictactoe_spec.rb

```diff
@@ -1,26 +1,117 @@
-require 'spec_helper' 
-require 'tictactoe' 
-describe TicTacToe do 
-  context 'when the board is viewed' do
-    it 'returns a single list with 9 elements' do 
-     game = TicTacToe.new
+require 'spec_helper'
+require 'tictactoe'
+## Faker Classes
```

Comment:
Does this comment add value? 

I don't want to you to get the impression that *all* comments are bad, but I think that the first thing to do when we are thinking about adding comments is to check if there is a way to encode that information within the code itself. The name of the class is `FakeIO`, which gives me a pretty good idea that this it is a fake object.

Just something to think about. Not a blocker.